deploy/helm/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: kubernetes-nmstate
+description: Declarative node network configuration driven through Kubernetes API.
+type: application
+version: 0.1.0
+appVersion: "1.16.0"
+home: https://github.com/nmstate/kubernetes-nmstate
+icon: https://nmstate.io/kubernetes-nmstate/assets/images/logo/fullcolor.png
+keywords:
+- nmstate
+- networking
+- NetworkManager
+maintainers:
+- email: ellorent@redhat.com
+  name: Enrique Llorente Pastora
+- email: phoracek@redhat.com
```

Comment:
@mkowalski would you mind if you were listed instead of me? It would better reflect the reality

Reply:
This will happen. It's just that it's not the only place where we should remove you (e.g. there is also the OLM manifest) so I would merge this one as-is and then I will handle the metadata update myself

Reply:
works for me